bootstrap.sh

```diff
@@ -19,8 +19,8 @@ curl "https://releases.hashicorp.com/terraform/1.5.4/terraform_1.5.4_linux_amd64
 sudo unzip ./terraform.zip -d /usr/local/bin
 
 # Install Kubectl and Helm CLIs
-wget -q https://dl.k8s.io/release/$KUBECTL_VERSION/bin/linux/amd64/kubectl -O /usr/local/bin/kubectl
-wget -q https://get.helm.sh/helm-$HELM_VERSION-linux-amd64.tar.gz -O - | tar -xzO linux-amd64/helm > /usr/local/bin/helm
+wget -q https://dl.k8s.io/release/$KUBECTL_VERSION/bin/linux/arm64/kubectl -O /usr/local/bin/kubectl
+wget -q https://get.helm.sh/helm-$HELM_VERSION-linux-arm64.tar.gz -O - | tar -xzO linux-arm64/helm > /usr/local/bin/helm
```

Comment:
helm deploys now run on the new `arm-small` queue, so swapping the binary architecture to `arm64` for compatibility.